plugins/processors/k8sdecorator/k8sdecorator_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package k8sdecorator
+
+import (
+	. "github.com/aws/amazon-cloudwatch-agent/internal/containerinsightscommon"
+	"github.com/aws/amazon-cloudwatch-agent/internal/structuredlogscommon"
+	"github.com/influxdata/telegraf/metric"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func TestDisableMetricExtraction(t *testing.T) {
+	tags := map[string]string{MetricType: TypeCluster}
```

Comment:
Would be better to table unit test here with other options too (e.g test default case for `DisableMetricExtraction`) since the others configuration such as (e.g PrefFullPodName, NodeName) are missing here.

Reply:
Updated to add a default case when this new flag isnt set at all. Lemme know if it looks good.